assets/bingo/lang/de_de.json

```
@@ -1,17 +1,364 @@
+	"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Die Erzeugung der Landschaft beginnt 5 Sekunden nach der Generierung der Bingo-Karte. \nBeachte bitte, dass dies Lag in der Lobby verursachen kann, was in einigen Fällen zu Verlangsamung der Funktionalität in der Lobby führen kann.",
+	"bingo.lobby.settings.pregen_behavior.turn_on": "Wechsel zu “Bei Spielstart”",
+	"bingo.lobby.settings.pregen_behavior.turn_off": "Wechsel zu “Bei Generierung”",
+
+	"bingo.lobby.menu.back": "Zurück",
+	"bingo.lobby.menu.back.hover": "Zurück zur vorherigen Seite",
+
+	"bingo.lobby.settings.strict_mode.sign.line1": "Server",
+	"bingo.lobby.settings.strict_mode.sign.line2": "Strikter Modus",
+	"bingo.lobby.settings.strict_mode.sign.line3": "",
+	"bingo.lobby.settings.strict_mode.sign.on": "Aktiviert",
+	"bingo.lobby.settings.strict_mode.sign.off": "Deaktiviert",
+	"bingo.lobby.settings.strict_mode.info": "Diese Einstellung ist für öffentliche Server. Bei dieser Einstellung sind alle in der Lobby im Abenteuermodus und die Spieleinstellungen sind deaktiviert für Nicht-Operatoren.",
+	"bingo.lobby.settings.strict_mode.turn_on": "Strikten Modus aktivieren",
+	"bingo.lobby.settings.strict_mode.turn_off": "Strikten Modus deaktivieren",
+	"bingo.lobby.settings.strict_mode.revoke_from_player": "Du kannst den OP Status eines bestimmten Spielers zurückziehen mit %s Kommando. Ersetze <player_name> mit dem Spieler dessen Status Du zurückziehen willst.",
+	"bingo.lobby.settings.strict_mode.revoke_from_player.click": "diesem",
+	"bingo.strict_mode.confirm_op": "Bitte %s um deine OP Status zu bestätigen.",
+	"bingo.strict_mode.confirm_op.click": "klick hier",
+
+	"bingo.lobby.tutorial.generate_chests.sign.line1": "",
+	"bingo.lobby.tutorial.generate_chests.sign.line2": "Truhen",
+	"bingo.lobby.tutorial.generate_chests.sign.line3": "erneut",
+	"bingo.lobby.tutorial.generate_chests.sign.line4": "erzeugen",
+
+	"bingo.lobby.tutorial.basics.sign.line1": "",
+	"bingo.lobby.tutorial.basics.sign.line2": "Minecraft Bingo",
+	"bingo.lobby.tutorial.basics.sign.line3": "Grundlagen",
+	"bingo.lobby.tutorial.basics.sign.line4": "",
+
+	"bingo.lobby.tutorial.basics1.sign.line1": "Minecraft Bingo",
+	"bingo.lobby.tutorial.basics1.sign.line2": "ist eine",
+	"bingo.lobby.tutorial.basics1.sign.line3": "Jagdquest",
+	"bingo.lobby.tutorial.basics1.sign.line4": "nach Gegenständen.",
+
+	"bingo.lobby.tutorial.basics2.sign.line1": "Eine Bingo-Karte",
+	"bingo.lobby.tutorial.basics2.sign.line2": "ist ein 5x5 Gitter",
+	"bingo.lobby.tutorial.basics2.sign.line3": "mit einem Gegenstand",
+	"bingo.lobby.tutorial.basics2.sign.line4": "pro Zelle.",
+
+	"bingo.lobby.tutorial.basics3.sign.line1": "Um eine Zelle",
+	"bingo.lobby.tutorial.basics3.sign.line2": "freizuschalten,",
+	"bingo.lobby.tutorial.basics3.sign.line3": "erhalte den dar-",
+	"bingo.lobby.tutorial.basics3.sign.line4": "gestellte Gegenstand.",
```

```suggestion
	"bingo.lobby.tutorial.basics3.sign.line1": "Um eine Zelle",
	"bingo.lobby.tutorial.basics3.sign.line2": "freizuschalten,",
	"bingo.lobby.tutorial.basics3.sign.line3": "erhalte das dar-",
	"bingo.lobby.tutorial.basics3.sign.line4": "gestellte Item.",
```
Unless you find a way to make it fit with "Gegenstand", but like this it doesn't
Also note that it should have been "dargestellte**n**", but it didn't fit either way.

I don't think it's a big deal to have the anglicism in there once, tho.

---

Fits only with "Force Unicode" :-(

---

Yeah, unicode font is smaller, but it's not the default